shell/imports/server/migrations.js

```diff
@@ -787,6 +787,49 @@ function setIpBlacklist(db, backend) {
   }
 }
 
+function notifyIdentityChanges(db, backend) {
+  // Notify users who might be affected by the identity model changes.
+  //
+  // Two types of users are affected:
+  // - Users who have multiple identities with differing names.
+  // - Users who have identities that are shared with other users.
+  //
+  // However, the second group seems like it must be a subset of the first group. So we only check
+  // for the first.
```

Comment:
Hmm. I wasn't really thinking about shared identities that are shared between accounts belonging to the same human, and hence have the same name as the user's other identities. I was thinking about brand accounts, which would presumably have a different name from the user's primary identity.

It seems to me, though, that the use case you described is largely unaffected by the change, or at least it's not terribly important to notify them in advance. They aren't at risk of accidentally revealing a secret identity when the change occurs.

So I'm not going to worry about detecting this case. (I know it wouldn't be super-hard but my time to work on sandstorm stuff is very limited...)